qiime2/plugin/plugin.py

```diff
@@ -233,6 +444,23 @@ def decorator(transformer):
             return decorator(_fn)
 
     def register_semantic_types(self, *type_fragments):
+        """Register semantic type fragments to the plugin
```

Comment:
Have you defined semantic type framgrments anywhere. I am not sure familar with this term.

Reply:
Nope. Hopefully the parameter section clarifies what that means a bit...

Reply:
It does. I wasn't sure if you wanted to address it before you used it in a sentence but I definitely understood before i was done with the doc string